atlante/template_helpers.go

```diff
@@ -5,42 +5,83 @@ import (
 	"errors"
 	"fmt"
 	"io"
-	"log"
 	"math"
 	"reflect"
-	"strconv"
 	"strings"
 	"text/template"
 	"time"
 
+	"github.com/gdey/as"
+	"github.com/prometheus/common/log"
+
 	"github.com/go-spatial/geom"
 	"github.com/go-spatial/geom/planar/coord"
 	"github.com/go-spatial/maptoolkit/atlante/server/coordinator/field"
+	"github.com/go-spatial/maptoolkit/atlante/template/grating"
 	"github.com/go-spatial/maptoolkit/atlante/template/remote"
 	"github.com/go-spatial/maptoolkit/atlante/template/trellis"
 )
 
 var funcMap = template.FuncMap{
-	"to_upper":     strings.ToUpper,
-	"to_lower":     strings.ToLower,
-	"format":       tplFormat,
-	"now":          time.Now,
-	"div":          tplMathDiv,
-	"add":          tplMathAdd,
-	"sub":          tplMathSub,
-	"mul":          tplMathMul,
-	"neg":          tplMathNeg,
-	"abs":          tplMathAbs,
-	"seq":          tplSeq,
-	"new_toggler":  tplNewToggle,
-	"rounder_for":  tplRoundTo,
-	"rounder3":     tplRound3,
-	"first":        tplFirstNonZero,
-	"DrawBars":     TplDrawBars,
-	"asIntSlice":   IntSlice,
-	"pixel_bounds": PixelBounds,
-	"join":         tplStringJoin,
-	"idx":          tplIndexOf,
+	"to_upper":           strings.ToUpper,
+	"to_lower":           strings.ToLower,
+	"format":             tplFormat,
+	"now":                time.Now,
+	"div":                tplMathDiv,
+	"add":                tplMathAdd,
+	"sub":                tplMathSub,
+	"mul":                tplMathMul,
+	"neg":                tplMathNeg,
+	"abs":                tplMathAbs,
+	"seq":                tplSeq,
+	"new_toggler":        tplNewToggle,
+	"rounder_for":        tplRoundTo,
+	"rounder3":           tplRound3,
+	"first":              tplFirstNonZero,
+	"DrawBars":           TplDrawBars,
+	"asIntSlice":         IntSlice,
+	"pixel_bounds":       PixelBounds,
+	"join":               tplStringJoin,
+	"split":              tplStringSplit,
+	"idx":                tplIndexOf,
+	"point":              tplNewPoint,
+	"SimpleGridFromArgs": simpleGridFromArgs,
```

Comment:
any reason for using a different naming convention for `SimpleGridFromArgs` from the other functions (with the exception of `DrawBars`)?

Reply:
I've been moving away from the tplNewPoint as they are all declared here. So, the tpl is redundent.